cmd/importer/main_test.go

```diff
@@ -0,0 +1,65 @@
+package main
```

Comment:
Don't need to add test for it.. since the usage is minimal. And these tests are like integration tests. We only need unit tests right now.